src/admin/resolvers/mutations/ScheduledItem.integration.ts

```diff
@@ -324,5 +442,112 @@ describe('mutations: ScheduledItem', () => {
         await eventTracker.getCall(0).args[0].scheduledCorpusItem.externalId
       ).to.equal(scheduledItem.externalId);
     });
+
+    it('should fail if user has read-only access', async () => {
+      const headers = {
+        name: 'Test User',
+        username: 'test.user@test.com',
+        groups: `group1,group2,no-access-for-you`,
```

Comment:
Same comment as above regarding the read only test

Reply:
Good catch, thank you!